css/style.css

```diff
@@ -758,6 +1264,59 @@
     color: #fff;
     font-size: 16px;
   }
+
+  .info__img {
+    height: auto;
+    width: 100%;
+    max-width: 767px;
+  }
+
+  .info__title {
+    font-size: 62px;
+    text-align: right;
+    padding: 0 100px 30px 0;
+  }
+
+  .wrap-information {
+    justify-content: center;
+    flex-direction: column;
+    height: 669px;
+    margin-top: -33px;
+  }
+
+  .red-first {
+    max-width: 767px;
+    height: 487px;
+    float: none;
+  }
+
+  .redrow {
+    flex-direction: row;
+    display: flex;
+    height: 300px;
+  }
+
+  .details {
+    max-width: 767px;
+    height: 487px;
+  }
+
+  .redrow-scnd {
+    flex-direction: row;
+    display: flex;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla-pro/pullRequest?prid=5839070)